src/Jackett.Common/Indexers/Shazbat.cs

```diff
@@ -158,27 +146,22 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
                         release.Guid = release.Link;
                         var qLinkComm = row.QuerySelector("td:nth-of-type(5) a:nth-of-type(2)");
                         release.Comments = new Uri(SiteLink + qLinkComm.GetAttribute("href"));
-
-                        var dateString = row.QuerySelector(".datetime").GetAttribute("data-timestamp");
+                        var dateString = row.QuerySelector(".datetime")?.GetAttribute("data-timestamp");
```

Comment:
this is odd. I would not expect the date to be missing. is this belts and braces coding, or did you come across a row with a missing date?

Reply:
This was because the next line was already checking for null before parsing the date result, which makes me think the previous maintainer might have come across it. The single page of HTML I had access to for testing was not missing any dates.